### PR TITLE
"Profiling your C# code" in c_sharp_basics.rst

### PR DESCRIPTION
**Pull request by van800**

### CODE REVIEW

File: getting_started/scripting/c_sharp/c_sharp_basics.rst [hunk 1]
```diff
@@ -259,3 +259,9 @@ msbuild to use, run:
 .. code-block:: none
 
     msbuild /t:restore
+
+Profiling your C# code
+-----------------------------
+
+- `Mono log profiler <https://www.mono-project.com/docs/debug+profile/profile/profiler/>`_ is available via `this PR <https://github.com/godotengine/godot/pull/26513>`_
+- External mono profiler like `JetBrains dotTrace <https://www.jetbrains.com/profiler/>`_ can be used as described `here <https://github.com/godotengine/godot/pull/34382>`_.
```
Comment (Calinou) [inline getting_started/scripting/c_sharp/c_sharp_basics.rst]: Capitalize Mono:

```suggestion
- External Mono profiler like `JetBrains dotTrace <https://www.jetbrains.com/profiler/>`_ can be used as described `here <https://github.com/godotengine/godot/pull/34382>`_.
```

Comment (van800) [reply getting_started/scripting/c_sharp/c_sharp_basics.rst]: Done

**Comment by van800** [comment]
Why this wasn't merged? It is better than nothing I guess. What it the plan?

**Comment by Calinou** [comment]
@van800 Sorry for the delay. I guess this is good to merge for now, we'll see if we can improve it later.

**Pull request merged**